api/dashboard/background_worker.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+import schedule
+import time
+import bittensor as bt
+from infrastructure.database import get_hotkey_collection
+from infrastructure.models import Hotkey, HotkeyType
+from neurons.remote_config import MinerConfig
+from neurons.storage import get_miners_metadata, get_validator_metadata
+from neurons.validators.validator import get_config
+

```

Comment:
create get_config tailored for dashboards needs, dont reuse that one from validator
dashboard is a seperate entity from miner or validator

Reply:
or instead just read ENV variables, get_config from validator requires registered hotkeys, and we dont want to do that

Reply:
Looking into it..

Reply:
Will use env vars so far and create config later if needed.